src/snmalloc/backend/globalconfig.h

```diff
@@ -70,11 +100,7 @@ namespace snmalloc
       key_global = FreeListKey(entropy.get_free_list_key());
 
       // Need to initialise pagemap.
-      Backend::init();
-
-#  ifdef USE_SNMALLOC_STATS
-      atexit(snmalloc::print_stats);
-#  endif
+      Pagemap::concretePagemap.init();
```

Comment:
Does the back end not need initialising anymore?  It might be nice to keep that path, even if it's an always-inline empty function, so that if the back end does need initialising in the future then we have a hook for doing it in place.

Reply:
The call was there before as the Backend was responsible for initialising the Backend. The Backend is a purely static class and has no state.  The LocalState is what might need initialising or things inside it, and that can be done by the Config.